sips/sips/sip-009-nft-standard.md

```diff
@@ -0,0 +1,85 @@
+## Preamble
```

Comment:
Hey @friedger, sorry for not getting to this sooner.  There's a couple of small formatting style changes that should be made here.

* The top-level section titles (Preamble, Abstract, etc.) should be H1, not H2.
* The Preamble section should not be a bullet list.  Each line should be its own paragraph.